### ISSUE TITLE
ZG related issues

### ISSUE THREAD

**Issue by malekoydaerb** [elysium-project/server#2520]
With the release of patch 1.11, several changes to ZG related things have occurred and are in error. 

Firstly, ZG flasks no longer persist through death. While this change to the ZG heart buff is correct, the change is currently in error in regards to the ZG flasks.  As per the archived patch notes on http://wowwiki.wikia.com/wiki/Patch_1.11.0 , there is no change to the effect of the ZG flasks. This change does happen, but from reading what is available, everything points to the change occurring closer to 1.12 proper. In addition, when the nerf occurred, blizzard changed the quest to cost only one honor token vs the current 3 as a means of compromise.

So, question to the devs: are we a patch 1.11 server or a 1.12 server now that naxx is here? Please pick one, because right now the zg flasks do not persist through death, yet still cost 3 honor tokens. Either restore the death persistence or lower the cost.

Second, ZG boss loot. At current all bosses in ZG drop two primary pieces of loot, this is in error. Quoting from the previously referenced patch notes, "After careful consideration we have decided to add an additional loot table to the High Priests of Zul'Gurub. Players should now see two items in addition to the tokens when slaying these bosses." As of this post, the High Priest bosses in ZG still are only dropping one loot item and one epic token, not the correct two items and one token. This needs corrected.

**Comment by Phoosy** [comment]
>So, question to the devs: are we a patch 1.11 server or a 1.12 server now that naxx is here? Please pick one, because right now the zg flasks do not persist through death, yet still cost 3 honor tokens. Either restore the death persistence or lower the cost.

Anathema is pending to become fully 1.12 content patch compliant within the week. There is no reason to adjust this for Anathema.
For Darrowshire and Elysium however, generally mechanics respect the 1.12 patch level mechanic but there have been exceptions made and there's no published logic as to where or why some mechanics respect this rule or not.

**Comment by ramangan** [comment]
Piggyback comment.
 **Anathema is pending to become fully 1.12 content patch compliant within the week** Can we expect all items, dropped in instances for example, added in 1.12 to be added to Anathema within the week?

**Comment by Phoosy** [comment]
I would presume the release of Naxx would coincide with the total update of all items to reflect their 1.12 stats, as all other patch releases also included updates of existing items if they were changed at that patch level.

**Comment by Lorilay2** [comment]
Both the reduction in cost and change to persisting through death were with 1.11, not 1.12. This bug report is regarding the missing reduction in cost and is unrelated to any 1.12 patch changes.

**Comment by malekoydaerb** [comment]
"Both the reduction in cost and change to persisting through death were with 1.11, not 1.12. This bug report is regarding the missing reduction in cost and is unrelated to any 1.12 patch changes."

are you sure? cause there was nothing in the 1.11 patch notes that even mentions this change. if it was an undocumented change, then fine, update the cost to the correct value and be done with it.

**Comment by Lorilay2** [comment]
Yes. It was not documented in any official patch notes but players mentioned it in July 2006 (patch 1.11).
